backend/tests/helpers/providers/record_provider.py

```diff
@@ -143,6 +143,12 @@ def data_record():
         return {
             "$schema": "http://localhost:5000/schemas/records/data.json",
             "_collections": ["Data"],
+            "titles": [{"title": fake.sentence()}],
+            "dois": [
+                {
+                    "value": RecordProvider.doi(),
```

Comment:
is dois required?

Reply:
if not, I wouldn't add it here. It should be added in the tests where it's needed.